src/averageAges.js

```diff
@@ -14,12 +16,27 @@
  *
  * @return {number}
  */
+
+// eslint-disable-next-line no-shadow
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter((person) => {
+    return century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm';
+  });
+
+  const lifespan = men.reduce((sum, age) => {
+    return sum + (age.died - age.born);
+  }, 0);
+
+  const averageAge = +(lifespan / men.length).toFixed(2);
```

Comment:
You use this logic not once. Create a separate function for calculating averageAge